frontend/src/components/forms/CreateEmployee.tsx

```diff
@@ -0,0 +1,301 @@
+                      placeholder="Enter first name"
+                      value={invitedFirstName}
+                      onChange={handleFirstNameChange}
+                      maxLength={50}
+                    />
+                    <FormErrorMessage>First Name is required.</FormErrorMessage>
+                  </FormControl>
+                </Box>
+
+                <Box w="46%">
+                  <FormControl isRequired isInvalid={invitedLastNameError}>
+                    <FormLabel>Last Name</FormLabel>
+                    <Input
+                      placeholder="Enter last name"
+                      value={invitedLastName}
+                      onChange={handleLastNameChange}
+                      maxLength={50}
+                    />
+                    <FormErrorMessage>Last Name is required.</FormErrorMessage>
+                  </FormControl>
+                </Box>
+              </Box>
+              <Box display="flex" flexDirection="row" mt="16px">
+                <FormControl isRequired isInvalid={invitedEmailError}>
+                  <FormLabel>SHOW Email</FormLabel>
+                  <Input
+                    placeholder="Enter SHOW email"
+                    value={invitedEmail}
+                    onChange={handleInvitedEmailChange}
+                    maxLength={254}
+                  />
+                  <FormErrorMessage>
+                    A valid email is required.
+                  </FormErrorMessage>
+                </FormControl>
+              </Box>
+            </Box>
+            <Box pt="16px" pb="16px">
+              <FormControl isRequired isInvalid={invitedAdminStatusError}>
+                <RadioGroup
+                  value={invitedAdminStatus}
+                  onChange={handleAdminStatusChange}
+                  pb="24px"
+                >
+                  <Flex>
+                    <Box marginRight="24px">
+                      <Radio size="lg" colorScheme="teal" value="1">
+                        <Text fontWeight="medium" color="#1B2A2C">
+                          Admin
+                        </Text>
+                      </Radio>
+                    </Box>
+                    <Box>
+                      <Radio size="lg" colorScheme="teal" value="2">
+                        <Text fontWeight="medium" color="#1B2A2C">
+                          Non Admin
+                        </Text>
+                      </Radio>
+                    </Box>
+                  </Flex>
+                  <FormErrorMessage>A selection is required.</FormErrorMessage>
+                </RadioGroup>
+              </FormControl>
+              <Checkbox
+                size="lg"
+                colorScheme="teal"
+                borderRadius="40px"
+                isChecked={isTwoFactorAuthenticated}
+                isDisabled={isDisabled}
+                onChange={(event) =>
+                  setIsTwoFactorAuthenticated(event.target.checked)
```

Comment:
I found an edge case where the checkbox remains checked when you switch to Admin from Non Admin. This causes `isTwoFactorAuthenticated` to be true when the form is submitted. 

![image](https://github.com/uwblueprint/supportive-housing/assets/43042601/858dff4c-666e-423e-aa5e-bc10d4b15ffa)

Reply:
good catch, I'll add in a fix